docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     deploy:
       resources:
         limits:
-          cpus: '0.9'
+          cpus: '4'
```

Comment:
why this upgrade?

Reply:
I did this change for my local, I'll revert. Thanks for noticing